health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -409,10 +428,54 @@
+        });
+
+        const updatedPlanEmployeeObjectForBoundaryInvalidate = {
+          ...fetchedPlanEmployeeForBoundaryInvalidate,
+          PlanEmployeeAssignment:
+            fetchedPlanEmployeeForBoundaryInvalidate?.PlanEmployeeAssignment?.length > 0
+              ? fetchedPlanEmployeeForBoundaryInvalidate.PlanEmployeeAssignment.map((employee) => ({
+                  ...employee,
+                  active: false,
+                }))
+              : [],
+        };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)